src/components/ContentNode/LinkableHeading.vue

```diff
@@ -9,24 +9,25 @@
 -->
 
 <template>
-  <component
-    :id="anchor"
-    :is="`h${level}`"
-  >
+  <div class="heading-wrapper">
```

Comment:
One downside to this restructuring of elements is that now you have to click the actual icon in order to get the link for the heading whereas before you could click anywhere within the title itself, which seemed like a better experience to me.

Is there a way of keeping that same behavior as before while also addressing the screen reader issue?

Reply:
I took another approach and I continued with the previous experience. So the change isn't too much https://github.com/apple/swift-docc-render/pull/698/commits/f9ce461bd83c90d5331108648b90b58d171abcf3